lib/node_modules/@stdlib/blas/ext/base/dsnannsumors/src/addon.c

```diff
@@ -41,13 +43,7 @@ static napi_value addon( napi_env env, napi_callback_info info ) {
 	STDLIB_NAPI_ARGV_STRIDED_FLOAT32ARRAY( env, X, N, strideX, argv, 1 );
 	STDLIB_NAPI_ARGV_STRIDED_FLOAT64ARRAY( env, Out, 2, strideOut, argv, 3 );
 
-	int io;
-	if ( strideOut < 0 ) {
-		io = -strideOut;
-	} else {
-		io = 0;
-	}
-
+	int64_t io = stdlib_strided_stride2offset( 2, strideOut );
 	double *out = Out;
```

Comment:
@headlessNode Why do we need `out` here? Why can we not just use `Out` directly?

Reply:
yeah `out` is redundant in this case. Thanks for pointing this out. I'll use the `Out` directly.